src/components/Checkbox/CheckBox.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { CheckBox } from './CheckBox';
+import { FormikFormWrapper } from '../FormikFormWrapper/FormikFormWrapper';
+
+it('should show CheckBox', () => {
```

Comment:
This is not accurate description + last test "should call render" is almost same and actually is better than this test.